lib/views/resetPassword.js

```diff
@@ -56,17 +56,47 @@ function resetPasswordPost (req, res) {
 
     return user.save()
   }).then((user) => {
-    sendResetPasswordMail(user)
+    return sendResetPasswordMail(user)
+  }).then(() => {
     if (!req.accepts('text/html')) {
-      return res.status(200).header('content-type', 'text/plain').send('Password reset successfully')
+      return res.status(200).header('content-type', 'text/plain').send('If a user was found associated with the specified e-mail address, a link to reset your password has been sent.')
     } else {
       res.send(pug.renderFile('templates/views/resetPasswordDone.jade', locals))
     }
   }).catch((err) => {
-    if (!req.accepts('text/html')) {
-      return res.status(500).header('content-type', 'text/plain').send(err)
+    if (err === {}) {
+      if (!req.accepts('text/html')) {
+        return res.status(200).header('content-type', 'text/plain').send('If a user was found associated with the specified e-mail address, a link to reset your password has been sent.')
+      } else {
+        res.send(pug.renderFile('templates/views/resetPasswordDone.jade', locals))
+      }
     } else {
-      res.send(pug.renderFile('templates/views/resetPasswordDone.jade', locals))
+      var error = err
+      var statusCode = 500
+      if (err.message) {
```

Comment:
Do you only have `err.response` when you have `err.message`?

Reply:
I'm not sure.  I just looked at what Sendgrid sent back, and I parsed it out.  However, I was nervous that I could not rely on the JSON structure to be fixed.